backend/api/appsettings.Development.json

```diff
@@ -22,5 +22,19 @@
     "Notifications": {
       "ThrowExceptions": true
     }
+  },
+  "Keycloak": {
+    "Authority": "https://dev.loginproxy.gov.bc.ca/auth/realms/standard",
+    "Audience": "pims-backend-4403",
+    "Secret": "[USE SECRETS]",
+    "Client": "pims-backend-4403",
+    "ServiceAccount": {
+      "Audience": "pims-backend-4403",
```

Comment:
It's odd that both the keycloak configuration is the same as the service account.  The service account is so that the API has authority to make requests to the CSS api.  The keycloak configuration is so that the api can validate tokens from users.